aptos-move/aptos-aggregator/src/aggregator_change_set.rs

```diff
@@ -0,0 +1,320 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    aggregator_extension::{
+        AggregatorID, DerivedFormula,
+    },
+    bounded_math::{SignedU128, code_invariant_error, BoundedMath, expect_ok}, delta_math::{DeltaHistory, merge_data_and_delta, merge_two_deltas},
+};
+use move_binary_format::errors::PartialVMResult;
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub enum AggregatorChange {
+    Data {
+        value: u128,
+    },
+    StringData {
+        value: Vec<u8>,
+    },
+    AggregatorDelta {
+        delta: SignedU128,
+        max_value: u128,
+        history: DeltaHistory,
+    },
+    SnapshotDelta {
```

Comment:
nit: should we add a comment for SnapshotDelta, to convey the delta is from the base_aggregator (and the correctness of the delta is checked by base aggregator)